modules/tinymce/src/core/test/ts/browser/content/EditorResetContentTest.ts

```diff
@@ -62,6 +62,7 @@ describe('browser.tinymce.core.content.EditorResetContentTest', () => {
       editor.resetContent();
       assert.isFalse(hasXssOccurred, 'XSS should not have occurred');
       (editor.getWin() as any)[xssFnName] = null;
+      McEditor.remove(editor);
```

Comment:
I would add it in an afterEach function for this context. This way it will run wether the tests fail or not.

Reply:
As discussed, I'm intentionally not removing the editor if the test fails